README.md

```diff
@@ -63,7 +63,43 @@ datetime | consumption
 
 To work on the challenge, please fork this repository or download it. After you have finished, you can send us a link to the fork or simply zip the repository and email it to us.
 
-### Questions?
+
+## The Frontend Challenge
+#### Setup
+The frontend challenge is located in the project folder `frontend`. To be able to use APIs you have to setup the dev environment:
+* Create the dev environment with `virualenv venv`.
```

Comment:
@kristjanstrojan Is python 3 necessary for this challenge?

Reply:
Mispelling:
```diff
- Create the dev environment with `virualenv venv`.
+ Create the dev environment with `virtualenv venv`.
```

Reply:
@hankehly not necessary, but prefered. Acctually our instructions say python 3.6. So better to be explicit like `virtualenv -p python3  venv`?